### ISSUE TITLE
Check if the Docker build is useful as it is

### ISSUE THREAD

**Issue by mikegerber** [OCR-D/ocrd_calamari#36]

**Comment by mikegerber** [comment]
Probably going to close/remove this. We already have two Docker builds (ocrd_all and ocrd-galley) and should concentrate on that. 

Pending review of things that need to be removed.

**Comment by mikegerber** [comment]
(And pending communication with @kba, but I doubt he will see this differently)

**Comment by bertsky** [comment]
I disagree. ocrd_all will soon switch from fat container to slim container, meaning that it will delegate to the images that its sub-modules provide themselves. These module images must then abide by the OCR-D convention (which will contain a network entrypoint) and be available in a public container registry like Dockerhub (and ideally backed by CI). So disabling Docker build is the wrong direction IMO.

**Comment by kba** [comment]
> (And pending communication with @kba, but I doubt he will see this differently)

I agree with @bertsky here, we would be disabling it now only to reenable it in a few weeks when we start providing slim containers (one Docker container per project with workers listening on queue).

**Comment by mikegerber** [comment]
Ah. Then I'll wait/switch to building a container according to OCR-D convention.

Is there documentation about the OCR-D convention?

**Comment by kba** [comment]
> Ah. Then I'll wait/switch to building a container according to OCR-D convention.
> 
> Is there documentation about the OCR-D convention?

Not for the new slim containers yet, since we require the Web API reference implementation in core to be ready which it is not but will be soon.

Once that's in place, we will update our [docker documentation](https://ocr-d.de/en/spec/docker)
